src/backend/gl/src/conv.rs

```diff
@@ -87,6 +76,70 @@ impl FormatDescription {
     }
 }
 
+pub const COMPRESSED_RGB_S3TC_DXT1_EXT: u32 = 0x83F0;
```

Comment:
I think it would be better to expose this on `glow` side.

Reply:
👍 opened a PR https://github.com/grovesNL/glow/pull/166